manifests/examples/kvm/stretch-virtio-partitions.yml

```diff
@@ -0,0 +1,46 @@
+---
+name: debian-{system.release}-{system.architecture}-{%Y}{%m}{%d}
+provider:
+  name: kvm
+  virtio_modules:
+  - virtio_pci
+  - virtio_blk
+bootstrapper:
+  workspace: /target
+system:
+  release: stretch
+  architecture: amd64
+  bootloader: grub
+  charmap: UTF-8
+  locale: en_US
+  timezone: UTC
+volume:
+  backing: raw
+  partitions:
+    type: gpt
+    boot:
+      filesystem: ext2
+      size: 1GiB
+    swap:
+      size: 128MiB
+    root:
+      filesystem: ext4
+      size: 8GiB
+    tmp:
+      mountopts:
+        - nodev
+        - noexec
+        - nosuid
+        - journal_ioprio=3
+      filesystem: ext4
+      size: 1GiB
+    var:
+      filesystem: ext4
+      size: 1GiB
+    var/tmp:
```

Comment:
My intension in first place was to not break the syntax anywhere.. For sure - this change would be wonderful. This would generally enhance the common comprehension of the configuration.